packer/build_with_docker.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+#
+# Copyright 2020 ScyllaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REALDIR=$(dirname $(readlink -f "$0"))
+DIR=$(dirname $(realpath -se $0))
+PDIRNAME=$(basename $(realpath -se $DIR/..))
+
+if [ "$PDIRNAME" = "aws" ] || [ "$PDIRNAME" = "gce" ] || [ "$PDIRNAME" = "azure" ]; then
```

Comment:
`build_with_docker.sh` reffer to building based on Centos. we don't support Azure based on at from day one. So it can be drop.

Reply:
@yaronkaikov Dropped azure from build_with_docker.sh.